pageserver/src/pgdatadir_mapping.rs

```diff
@@ -1465,7 +1470,30 @@ impl<'a> DatadirModification<'a> {
         content: &[u8],
         ctx: &RequestContext,
     ) -> anyhow::Result<()> {
-        let policy = self.tline.get_switch_aux_file_policy();
+        let switch_policy = self.tline.get_switch_aux_file_policy();
+
+        let policy = {
+            let current_policy = AuxFilePolicy::int_to_opt(
+                self.tline
+                    .last_aux_file_policy
+                    .load(std::sync::atomic::Ordering::SeqCst),
```

Comment:
What's the rationale behind seqcst?

Reply:
Just use the strongest guarantee b/c the code path is not critical and avoids potential problems with atomic orderings 🤣